studies/calgames_kinematics/src/test/java/org/team100/KinematicsTest.java

```diff
@@ -11,21 +11,21 @@ public class KinematicsTest {
     @Test
     void testForward() {
```

Comment:
add more test cases, especially covering inputs that are confusing or invalid.  for example, if you give the inverse() method a pose which is outside the envelope that the mechanism can reach, what happens?  what __should__ happen?